it/deployment_ready_test.go

```diff
@@ -0,0 +1,172 @@
+package it
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	cond_v1 "github.com/atlassian/ctrl/apis/condition/v1"
+	"github.com/atlassian/smith"
+	smith_v1 "github.com/atlassian/smith/pkg/apis/smith/v1"
+	smith_testing "github.com/atlassian/smith/pkg/util/testing"
+	"github.com/stretchr/testify/assert"
+	apps_v1 "k8s.io/api/apps/v1"
+	core_v1 "k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func constructBundle(t *testing.T, progressDeadlineSeconds int32, containerParams ...string) *smith_v1.Bundle {
+	t.Parallel()
```

Comment:
I think it'd be better to move this to each test as first line instead of hiding side effects in this helper function =)

Reply:
Done